generators/app/index.js

```diff
@@ -249,6 +265,7 @@ module.exports = yeoman.Base.extend({
         options: {
           mainModule: true,
           ionicCss: this.answers.ionicCss,
+          jade: this.answers.jade,
```

Comment:
This isn't used later on, is it? As far as I can see, the jade answer is read from the .yo-rc instead as an option for the subgenerator.